zerver/views/documentation.py

```diff
@@ -131,12 +131,14 @@ def get_context_data(self, **kwargs: Any) -> Dict[str, Any]:
         if self.path_template == "/zerver/help/%s.md":
             context["page_is_help_center"] = True
             context["doc_root"] = "/help/"
+            context["doc_root_title"] = "Help center"
```

Comment:
Do we want to rename the "User documentation" top-level heading for the page to "Help center" as well?

Reply:
Yeah, I like "Help center" as the canonical name for these pages that we should consistently use everywhere.

Reply:
Agreed! :)